code/src/alpha_comp_extractor.cpp

```diff
@@ -0,0 +1,73 @@
+#include <alpha_comp_extractor.hpp>
+#include <data_extractor.hpp>
+
+namespace r2d2::thermal_camera {
+    alpha_comp_extractor_c::alpha_comp_extractor_c(mlx90640_i2c_c &bus, int row,
+                                                   int col)
+        : mlx_extractor_c(bus), row(row), col(col) {
+    }
+
+    void alpha_comp_extractor_c::extract(mlx_parameters_s &params) {
+        int data;
+
+        data = bus.read_register(registers::EE_ALPHA_ACC_SCALE);
+        const int alpha_scale =
+            data_extractor_s::extract_data(data, 0xF000, 12) + 30;
```

Comment:
Please give a clear name to your magic numbers or explain them in a comment.

Reply:
I'll only leave this comment here, but the same goes for your other uses of magic numbers.

Reply:
> 
> 
> Please see the pipeline `mlx90640_processor_c::get_temperature_pixel`. Each object is (approximately) responsible for its own calculation. Above the creation of each object in `mlx90640_processor_c::get_temperature_pixel`, there is a reference to a section in the datasheet, in which these numbers come back.